### ISSUE TITLE
Look at publishing test results in jobs with all pkgs

### ISSUE THREAD

**Issue by MatthijsBurgh** [tue-robotics/tue_robocup#1329]
Example https://github.com/cartographer-project/cartographer_ros/blob/master/azure-pipelines.yml

**Comment by PetervDooren** [comment]
So this would be equivalent to testing the entire system rather than components correct?

**Comment by MatthijsBurgh** [comment]
Not really, just uploading the test results to azure, so it is visible in the gui of the website
